app/javascript/admin/controllers/confirmation_modal_controller.js

```diff
@@ -19,6 +19,7 @@ export default class ConfirmationModalController extends ModalController {
   handleRecord({ endpoint, id, outcome }) {
     switch (endpoint) {
```

Comment:
I'm afraid this `switch` will get pretty unwieldy as we add cases. I wonder if at one case we should do something like the following:

1. Track endpoints that need the less common flow (e.g. redirect) in a constant array/set.
2. In this method, check if the endpoint is included in that constant. If yes, perform the redirect flow, if not perform the other flow. 

But this can definitely wait a bit longer.

Reply:
Please re-review @citizen428 🙏🏾